frontend/src/components/RestaurantDetails/RestaurantInfo.js

```diff
@@ -24,15 +27,16 @@ class restaurantInfo extends React.Component {
     };
 
     componentDidMount() {
-        fetch('http://localhost:3000/restaurant.json')
+        fetch('http://localhost:6543/restaurant/'+ this.props.url.match.params.id)
             .then(response => response.json())
-            .then(data => this.setState({restInfo: data.data[this.props.url.match.params.id-1]}))
+            .then(data => this.setState({restInfo: data.data[0][0]}))
```

Comment:
It's not clear for me, why do you get first element twice `data[0][0]`. What it he format of the data? According to controller it should be a one level list, so `data[0]` should be enough.